README.md

```diff
@@ -1,75 +1,47 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# E3 Broadcast Simulation
 
-# Project Name
-
-> One paragraph statement about the project.
+> Website simulating the E3 conference of 2021
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML-CSS-JAVASCRIPT
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Visit Live Demo Link](https://felipeg005.github.io/Capstone-html-css-Javascript/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+To get a local copy up and running follow these simple example steps
 
 ### Prerequisites
 
-### Setup
-
-### Install
+- Browser (Chrome,Firefoxe,IE,Opera,Safari)
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- Open Html.index file with your browser
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-## 🤝 Contributing
+👤 **Author**
 
-Contributions, issues, and feature requests are welcome!
+**Juan Felipe Gomez Tobon**
 
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@githubhandle](https://github.com/Felipeg005/)
+- Twitter: [@twitterhandle](https://twitter.com/JuanFGT05)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-felipe-gomez-tobon/)
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Thanks to Microverse for make it possible!
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License. Please add this, is important to give the credits for authors. 🐶

Reply:
Yeah! of course, I forgot it.  It's done!